### ISSUE TITLE
Stuck at "Trying to write tags" on MP4

### ISSUE THREAD

**Issue by hroman73** [mdhiggins/sickbeard_mp4_automator#837]
I have convert-mp4 set to false so when i point manual.py to a directory with a mix of mkv and mp4 it gets stuck at "Trying to write tags" on an MP4 file although its supposed to skip it.

**Comment by mdhiggins** [comment]
Its not converting its just writing tags, that's usual behavior
Why the file is getting stuck on trying to write tags is another issue, if you don't want tagging you can turn that feature off

**Comment by rpatterson** [comment]
I'm seeing this too.  I want it to tag existing MP4s, so this isn't about turning off that feature for me.

Where it gets stuck seems to be at an I/O system call.  `Ctrl-C` doesn't work to kill the process, I have to do `End process` through the Task Manager (Windows) and it takes a while after doing that for the process to exit.  I ran `python -m trace --trace manual.py ...` and the last output seems to be a file operation.

I'm reasonably certain I've seen it successfully tag other existing MP4s so it seems to be specific to certain files.  I've also been able to reproduce the hang with multiple existing MP4s in the same folder downloaded from the same source, so it's probably specific to a certain way of doing things.  The files that tagging hangs on all play just fine.

That's all the info I have, hope it helps!
